packages/dapp/pages/styles/ubiquity.css

```diff
@@ -44,6 +44,13 @@ button:hover,
 input:hover {
   background: #ffffff20;
 }
+button[disabled="disabled"],
```

Comment:
The disable style will apply to all buttons, not just the `claim` button. This can be seen in the `stake` button.